### PR TITLE
feat(integrations): add support for Buildium

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Add support for buildium

## Issue ticket number and link
[EXT-180](https://linear.app/nango/issue/EXT-180/add-support-for-buildium)

### CODE REVIEW

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -733,6 +733,34 @@ builder-io-public:
...
+    connection_config:
+        clientId:
+            type: string
+            title: Client ID
+            description: Your API Key Client ID
+    credentials:
+        apiKey:
+            type: string
+            title: API Key
+            description: The API key for your Buildium account
```
Comment (TBonnin) [inline packages/shared/providers.yaml:764]: there is new attributes like pattern or example? can you add them here.

Comment (hassan254-prog) [reply packages/shared/providers.yaml:764]: updated [with](https://github.com/NangoHQ/nango/pull/2818/commits/ab9594d098ce6ef2c3598ef1eb44aa56545968bc), picked an example of both from their API documentation

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -755,11 +755,15 @@ buildium:
...
             description: Your API Key Client ID
+            pattern: '^[a-z0-9_-]+$'
+            example: a228f0e7-b4a3-4150-b9ae-8552fc2880d3
     credentials:
         apiKey:
             type: string
             title: API Key
             description: The API key for your Buildium account
+            pattern: '^[a-z0-9_-]+$'
+            example: uOq2p+xlgpFdijfV/HqY+EvYpZKHRwlyhGuAVbJIxXs
```
Comment (TBonnin) [inline packages/shared/providers.yaml:766]: the regex doesn't match the value. It contains `+` or `/` for instance. You probably want `^[A-Za-z0-9+/]+$` instead

**Review by TBonnin — CHANGES_REQUESTED** [review]

File: packages/shared/providers.yaml (hunk 2 above)
Comment (hassan254-prog) [reply packages/shared/providers.yaml:766]: aah yes, updated with [commit](https://github.com/NangoHQ/nango/pull/2818/commits/457afb07627c43917bf969a1cdaca95a6081af97)

**Review by TBonnin — APPROVED** [review]

**Pull request merged**